db/schema.rb

```diff
@@ -420,6 +420,11 @@
   end
 
 
+  create_view "pg_stat_statements_info", sql_definition: <<-SQL
```

Comment:
Honestly not sure why this was added and seems like it's breaking specs possibly. Will try to remove.